src/sentry/notifications/notification_action/action_handler_registry/sentry_app_handler.py

```diff
@@ -33,10 +33,12 @@ class SentryAppActionHandler(ActionHandler):
 
     data_schema = {
         "$schema": "https://json-schema.org/draft/2020-12/schema",
+        "description": "The data schema for a Sentry App Action",
         "type": "object",
         "properties": {
             "settings": {"type": ["array", "object"]},
         },
+        "required": ["settings"],
```

Comment:
I no longer feel confident this should be required - we have actions in the db without data. Are those invalid? If so what should we do about them?